modules/fluid_properties/include/interfaces/VaporMixtureInterface.h

```diff
@@ -0,0 +1,68 @@
+#pragma once
```

Comment:
So is `VaporMixtureInterface` just a move from THM? Any changes? So there will need to be fixes to all the downstream apps?

Reply:
Hang on, all of this stuff came from THM? I was wondering why it seemed familiar...

Reply:
> So there will need to be fixes to all the downstream apps?

Yes...